Middleware/SVGs/PlanetNineLogo.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+
+public class PlanetNineLogo
+{
+    private readonly IWebHostEnvironment _env;
+    private readonly Lazy<Task> _initializationTask;
+    private string svg;
+    private string originalSVG;
+
+    public PlanetNineLogo(IWebHostEnvironment env)  
+    {
+        _env = env;
+        _initializationTask = new Lazy<Task>(() => DoInitializeAsync());
+    }
+
+    public async Task InitializeAsync() 
+    {
+        await _initializationTask.Value;
+    }
+
+    public async Task DoInitializeAsync() 
+    {
+        svg = await ReadFileFromWebRootAsync(_env, "planet-nine-logo.svg"); 
+        originalSVG = svg;
+    }
+
+    public async Task<string> SVG() 
+    {
+         svg = originalSVG;
+
+         Starfield starfield = new Starfield();
+         string starfieldSVG = await starfield.SVG("{{contents}}", _env, "100", "60", "#100a12", 62);
+
+         PlanetNine planetNine = new PlanetNine();
```

Comment:
C# allows for nicer instantiation..
Either do:

var planetNine = new PlanetNine();
Or
PlanetNine planetNine = new();

Keeps the repetitive declarations to a minimum.